docker/Dockerfile

```diff
@@ -28,3 +29,5 @@ RUN mkdir -p /data /backup
 USER pliers
 RUN python -m pliers.support.download
 WORKDIR /work
+# Just a test to guarantee correct installation
+RUN python -m pliers
```

Comment:
@yarikoptic - Would `RUN python -m pliers.support.download` not already test that the installation is correct?

Reply:
It is ran within source code base. In principle local module shouldn't be picked up, but I am paranoid, so thought it wouldn't hurt to test from a work dir

Reply:
But thinking about it, -m pliers shouldn't be correct if there is no `__main__` there... Let me check later...

Reply:
damn me -- I have managed to miss (was running build on another box) that it did fail as then I feared:
```
Step 11/11 : RUN python -m pliers
 ---> Running in c5405074bfef
/usr/local/bin/python: No module named pliers.__main__; 'pliers' is a package and cannot be directly executed
The command '/bin/sh -c python -m pliers' returned a non-zero code: 1
```
I will just remove that "Test" change for now.  Ideally it should just run all the tests upon build to verify correct operation in the environment it has created.  but let's keep it a separate issue ;)